packages/bcer-retailer-app/app/src/components/form/inputs/BusinessLocationInputs.tsx

```diff
@@ -29,65 +38,138 @@ const useStyles = makeStyles({
   radioWrapper: {
     padding: '0px 20px 15px 0px'
   },
-  optionalField: {
+  autocompleteField: {
+    '& .MuiAutocomplete-inputRoot': {
+      padding: '0px 12px 0px 0px !important'
+    }
+  },
+  helpIcon: {
+    fontSize: '22px',
+    color: '#0053A4'
+  },
+  tooltip: {
+    backgroundColor: '#0053A4',
+    fontSize: '14px'
+  },
+  arrow: {
+    color: '#0053A4'
   }
 })
 
-function BusinessLocationInputs() {
+function BusinessLocationInputs({formikValues, formikHelpers }: {formikValues: IBusinessLocationValues, formikHelpers: FormikHelpers<IBusinessLocationValues>}) {
   const classes = useStyles();
   const { values } = useFormikContext<IBusinessLocationValues>();
+  const [ predictions, setPredictions ] = useState<Array<BCGeocoderAutocompleteData>>([]);
+  const [ autocompleteOptions, setAutocompleteOptions ] = useState<Array<string>>([]);
+  const [{ data, error, loading }, getSuggestions] = useAxiosGet('', { manual: true })
+  // const {
+  //   placesService,
+  //   placePredictions,
+  //   getPlacePredictions,
+  //   isPlacePredictionsLoading, 
+  // } = usePlacesService({
+  //   apiKey: process.env.GOOGLE_MAPS_API_KEY
+  // })
+
+  useEffect(() => {
+    if (data && data.features?.length > 0) {
+      console.log(data.features)
+      setPredictions(data.features)
+      setAutocompleteOptions(data.features.map((e: BCGeocoderAutocompleteData) => e.properties.fullAddress))
+    }
+  }, [data])
+
+  const handleAutocompleteSelect = ( value: any, reason: AutocompleteChangeReason, details?: AutocompleteChangeDetails<any>) => {
+    const fullLocation = predictions.find(e => e.properties.fullAddress === value)
+    formikHelpers.setFieldValue('addressLine1', fullLocation ? fullLocation.properties.fullAddress : '')
+    formikHelpers.setFieldValue('precision', fullLocation.properties.precisionPoints)
+    formikHelpers.setFieldValue('city', fullLocation.properties.localityName)
+  }
+  
+  const getAutocomplete = (e: any) => {
+    getSuggestions({url: `https://geocoder.api.gov.bc.ca/addresses.json?minScore=50&maxResults=5&echo=false&autoComplete=true&brief=false&matchPrecision=occupant,unit,site,civic_number,block&addressString=${e.target.value}`})
```

Comment:
I see there is no api key here but wondering if we should keep the link in env for better organization